addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_document.py

```diff
@@ -411,10 +418,26 @@ def _get_recipient_values(self, partner, is_simplified=False):
 
         return {'recipient': recipient_values}
 
+    def _get_refunded_values(self, values):
+        if not values.get('is_refund'):
+            return {}
+        refunded_doc = values['refunded_doc']
+        refunded_name = values['refunded_name']
+        if refunded_doc:
+            sequence, number = refunded_doc._get_tbai_sequence_and_number()
+        else:
+            sequence, number = self._get_tbai_seq_from_name(refunded_name)
+        return {
+            'refunded_serie': sequence,
+            'refunded_num': number,
+            'refunded_date': values['refunded_doc_invoice_date'],
```

Comment:
Not sure whether it is needed. But it was like this previously in the XML
```suggestion
            'refunded_date': refunded_doc.xml_attachment_id and refunded_doc._get_tbai_signature_and_date()[1] or values['refunded_doc_invoice_date'],
```